_search-plugins/neural-search.md

```diff
@@ -196,7 +224,135 @@ GET my_index/_search
   }
 }
 ```
+{% include copy-curl.html %}
 
+The response contains the matching document:
 
+```json
+{
+  "took" : 36,
+  "timed_out" : false,
+  "_shards" : {
+    "total" : 1,
+    "successful" : 1,
+    "skipped" : 0,
+    "failed" : 0
+  },
+  "hits" : {
+    "total" : {
+      "value" : 1,
+      "relation" : "eq"
+    },
+    "max_score" : 1.2251667,
+    "hits" : [
+      {
+        "_index" : "my-nlp-index",
+        "_id" : "1",
+        "_score" : 1.2251667,
+        "_source" : {
+          "passage_text" : "Hello world",
+          "id" : "s1"
+        }
+      }
+    ]
+  }
+}
+```
+
+### Setting a default model on an index or field
+
+To eliminate passing the model ID with each neural query request, you can set a default model on a k-NN index or a field. 
 
+First, create a [search pipeline]({{site.url}}{{site.baseurl}}/search-plugins/search-pipelines/index/) with a [`neural_query_enricher`]({{site.url}}{{site.baseurl}}/search-plugins/search-pipelines/neural-query-enricher/) request processor. To set a default model on an index, provide the model ID in the `default_model_id` parameter. To set a default model on a specific field, provide the field name and the corresponding model ID in the `neural_field_default_id` map. If you provide both `default_model_id` and `neural_field_default_id`, `neural_field_default_id` takes precedence:
```

Comment:
To set a default model "for" an index? To set a default model "for" a specific field?